pkg/provider/gcp.go

```diff
@@ -332,7 +332,7 @@ const (
 func (gcp *GCPProvider) Preflights() []*Preflight {
 	return []*Preflight{
 		{Name: string(PreflightCheckEnabledServices), Callback: gcp.validateEnabled},
-		{Name: string(PreflightCheckServiceAccountPermissions), Callback: gcp.validatePermissions},
+		// {Name: string(PreflightCheckServiceAccountPermissions), Callback: gcp.validatePermissions},
```

Comment:
FYI: https://github.com/pluralsh/plural-cli/pull/647 removes single check that you have mentioned.